features/0592-indy-attachments/README.md

```diff
@@ -110,16 +110,18 @@ This is the format emitted by libindy's [indy_issuer_create_credential()](https:
 
 ```jsonc
 {
-"schema_id": "4RW6QK2HZhHxa2tg7t1jqt:2:bcgov-mines-act-permit.bcgov-mines-permitting:0.2.0",
-"cred_def_id": "4RW6QK2HZhHxa2tg7t1jqt:3:CL:58160:default",
-"rev_reg_def_id", "EyN78DDGHyok8qw6W96UBY:4:EyN78DDGHyok8qw6W96UBY:3:CL:56389:CardossierOrgPerson:CL_ACCUM:1-1000",
-"values": {
-    "attr1" : {"raw": "value1", "encoded": "value1_as_int" },
-    "attr2" : {"raw": "value1", "encoded": "value1_as_int" }
-},
-// Fields below can depend on Cred Def type
-"signature": <signature>,
-"signature_correctness_proof": <signature_correctness_proof>
+    "schema_id": "4RW6QK2HZhHxa2tg7t1jqt:2:bcgov-mines-act-permit.bcgov-mines-permitting:0.2.0",
+    "cred_def_id": "4RW6QK2HZhHxa2tg7t1jqt:3:CL:58160:default",
+    "rev_reg_id", "EyN78DDGHyok8qw6W96UBY:4:EyN78DDGHyok8qw6W96UBY:3:CL:56389:CardossierOrgPerson:CL_ACCUM:1-1000",
+    "values": {
+        "attr1" : {"raw": "value1", "encoded": "value1_as_int" },
+        "attr2" : {"raw": "value1", "encoded": "value1_as_int" }
```

Comment:
Tweak - should be `value2`

Reply:
good catch, similarly for the rest: incoming